Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: test
+test:
+	go test -v .
+
+.PHONY: lint
+lint:
+	golangci-lint run
```

Comment:
can we have .golangci.yml like in other projects ?

e.g:

https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/master/.golangci.yml